carma_ros2_utils/include/carma_ros2_utils/containers/containers.hpp

```diff
@@ -46,9 +49,19 @@ std::vector<T, A> downsample_vector(const std::vector<T, A>& input, unsigned int
 
   output.reserve((input.size() / n) + 1);
 
+  bool included_last_point = false;
```

Comment:
Could we consider altering the name of this variable so that it doesn't get confused with `include_last_point` upon a quick glance, and also to help avoid any issue due to a small typo if this ever gets refactored? Maybe `has_included_last_point` or something similar?

Reply:
renamed